fixed/added images (#61)
asetalias/blog: content/post/Android_Root101.en.md

+++
title = "A Guide to Flashing Custom ROMs [Android]"

> slug = "custom-rom-101"

Reviewer: ```suggestion
slug = "slapping-custom-roms-on-android-like-a-pro"
```
Just a suggestion :heart: 
And this was a joke, where I meant. Make descriptive slugs.

Author: 👅 that slapping 😆

date = "2018-02-25T00:00:00"

> image = "img/rohanvtk/csrom.jpg"

Reviewer: Since we are having this change in many place, 
Can you try and put a URL of this image rather than the image itself. Would help a ton to save bits of space. Thanks,

Author: I reduced the size to 78KB. All my images are now under 90KB. 
💛 #optimized 😄

Reviewer: :hatching_chick:

> description = "Sick and tired of the bloatware on your current OEM ROM. Head over and learn how to flash custom ROMs."

Reviewer: Somewhere I think this blog would be a little better for new people if we can have one line definitions for words and annotations used in the blog. Sure, people can Google that stuff but they would then have no reason to read a 101 guide.

Author: Sure! I'll work on it. I realize you're 💯 correct about this.

Author: @Sc0rpi0n101

Reviewer 2: Yeah, that was also my point xD
along with some others :rofl: 
Waiting for an update then

author = "RohanVTK"
tags = ['Smartphone', 'Customization','Android']
categories = ['Android']
+++

> {{<redwarntxt>}}This blog starts off with a warning: Although the process is super easy and the chances of your phone getting bricked(useless) are negligible but, it's my responsibility to make you aware of it{{</redwarntxt>}}

Reviewer 3: For stuff like this probably you should also consider adding images. That show how it looks now.

Author: I want to add images. Is there a way to resize images in hugo?
The current images are going way off the limits. 😭

Reviewer: Same as we add images in markdown as Hugo uses Blackfriday Markdown parser 
`![](IMAGE_URL)`

For resizing images, you can try [these hacks](https://gist.github.com/uupaa/f77d2bcf4dc7a294d109) if you are using Markdown. (Do test them on your local deployment first)
Or just sprinkle some HTML.

![](https://cultofthepartyparrot.com/parrots/hd/gentlemanparrot.gif)

Reviewer: Party parrot complimentary with every PR 

:hatching_chick: <<<<< Party Parrot

Author: I created a shortcode file for that. CSS helped 😄 But, not center aligned. 😢

Reviewer: @Sc0rpi0n101 Didn't you have all images as center aligned.

Reviewer 2: not all, I provided an option to turn images center-aligned using `#center`

Author: @Sc0rpi0n101 using the shorthand method?

Reviewer 2: `[](../abc.jpg#center)`

* Now open up the command prompt on Windows with **_adminitrator priviliges_**.

    _(To do that, press the `Windows key` on your keyboard and search 'cmd', right click when command prompt comes up and chose 'Run as Administrator')_

* Now go the directory where the files you downloaded have been stored.
    if you don't know how to use Command Prompt

> > `cd <dirname>` is use to go up a directory

Reviewer 2: Come on, we talked about this
The file structure is not linear, there is no going up a directory
Doesn't work that way

```suggestion
> `cd <dirname>` is used to go to the directory named "<dirname>"
```

> `cd ..` is used to go down one directory

> Enable Developer Options on your phone and enable USB-Debugging. To know more [Google Search](https://www.google.com)

Reviewer 2: Woah
```
(https://www.google.com) ??
```
Also, better to link an article than a google search
If people have to go out of the article to search for related stuff, why would they read your article? there are a lot more articles available which describe everything.
**Never** make your audience leave your page

Finally, enter fastboot mode on your phone. How-to methods have been listed below(Use any one of the following):

1. Reboot to Bootloader ->(Hold Power + Volume down)    {*_For Xiaomi Devices_*}

> 2. Connect your phone to the PC after enabling the ADB debugging mode in the developer settings

Reviewer 2: Did you explain how to do that?
Again, the same thing.
If I have to go search how to do things you have mentioned, Why would I read your article?
I'd go find a better one

_Reboot the Device._

> {{<yellowborder>}}Congratulations! Now you have a phone with a CUSTOM ROM of your own choice with no bloatwares.{{</yellowborder>}}

Reviewer 3: Again some reference images needed.

Author: Added two images. Next blog me I'll add a lot of descriptive images. Pakka Promise! 😄

Reviewer 2: You know he means adding images in the PR discussion for what this stuff actually looks like after rendering as we cannot guess everything from the code

He's not talking about adding images to the blog :rofl:

______

*If during the process you encoutered any issues, do look up your issues on XDA or else Google your issues, you'll surely find a solution.*

> *Want to connect?* [📱Connect Here](https://twitter.com/RohanVTK).

Reviewer: What I actually meant :laughing: 
```suggestion
*Want to connect?* [📱 Hit me up on Twitter @RohanVTK](https://twitter.com/RohanVTK)
```
1. Give them a reason to click the link
2. Give them an idea of what they are clicking without looking at the bottom left of their screen.

_Reboot the Device._

{{<yellowborder>}}Congratulations! Now you have a phone with a CUSTOM ROM of your own choice with no bloatwares.{{</yellowborder>}}

______

> *If during the process you encoutered any issues, do look up your issues on XDA or else Google your issues, you'll surely find a solution.*

Reviewer 2: Woah, did you just tell me "**If you find any issues in the process I taught you,  go fix them yourself**" ?
Because that's exactly what it means.

It can easily be fixed. All you need is a very small change.
